http-client/mhs/asynchronous-express-message-pattern/Return-information-from-ebxml-fault-returned-from-spine-in-original-post-request-when-sync-async-requested.http

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
How did you named those files from?
I would expect someting like "async_reliable_something.http"